README.md

```diff
@@ -58,90 +58,57 @@ Write [a config file](#Configuration) directly, or generate it with
 
 Then start the `xremap` daemon by running:
 
-```
+```sh
 sudo xremap config.yml
 ```
 
-(You will need to leave it running for your mappings to take effect.)
+You might need to run it with sudo for application-specific remaps to work in
```

Comment:
question: What does this mean? I think it's generally easier to make application-specific remaps work without sudo. Sudo makes everything but application-specific remaps easier.